crates/graphql-federated-graph/src/render_sdl/directive_definition.rs

```diff
@@ -0,0 +1,34 @@
+use super::input_value_definition::display_input_value_definition;
+use crate::{federated_graph::*, Directive, FederatedGraph};
+use std::fmt;
+
+pub(crate) fn display_directive_definition(
+    directive_definition: &DirectiveDefinition,
+    directives_filter: fn(&Directive, graph: &FederatedGraph) -> bool,
+    graph: &FederatedGraph,
+    f: &mut fmt::Formatter<'_>,
+) -> fmt::Result {
+    f.write_str("directive @")?;
+    f.write_str(&graph[directive_definition.name])?;
+
+    if directive_definition.arguments.1 > 0 {
```

Comment:
Something like `if !graph[directive_definition.arguments].is_empty()` would be clearer IMHO